python/nav/web/templates/status2/status.html

```diff
@@ -30,7 +29,12 @@
     {# Form for filtering events #}
     <a href="javascript:void(0);" class="button small secondary toggle-panel"></a>
     <div id="status-panel" class="hidden panel">
-      {% crispy form %}
+      {% if form.attrs %}
+        {% include 'custom_crispy_templates/flat_form.html' %}
+      {% else %}
+        {{ form }}
+      {% endif %}
+    <hr>
```

Comment:
The only difference here is that the `<hr>` is not within the form, but visually the outcome is the same, so I would not care